crates/core/src/lib.rs

```diff
@@ -53,6 +58,28 @@ impl Config {
     pub fn wasmtime_config(&mut self) -> &mut wasmtime::Config {
         &mut self.inner
     }
+
+    /// Enable the Wasmtime compilation cache. If `path` is given it will override
+    /// the system default path.
+    ///
+    /// For more information, see the [Wasmtime cache config documentation][docs].
+    ///
+    /// [docs]: https://docs.wasmtime.dev/cli-cache.html
+    pub fn configure_cache(&mut self, config_path: &Option<PathBuf>) -> Result<()> {
```

Comment:
Maybe `enable_cache`? `configure_cache(None)` seems like it should disable cache.